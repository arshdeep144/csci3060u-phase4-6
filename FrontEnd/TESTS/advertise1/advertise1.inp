login
user02
advertise
item01
999.99
100
logout
