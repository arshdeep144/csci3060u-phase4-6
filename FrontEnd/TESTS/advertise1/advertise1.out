Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter the name of the item
Please enter price of the item
Please enter the number of days until the auction ends
Welcome to the main menu. Please enter a command to continue: 
